### ISSUE TITLE
Parse feet/inches format

### ISSUE THREAD

**Issue by svenstaro** [Qalculate/libqalculate#89]
I convert freedom units to metric ones often and as such it'd be convenient if libqalculate understood `5'20` and `5'20"` which it currently does not appear to do.

**Comment by hanna-kn** [comment]
I agree that it would be nice support such format, and I've considered it before, but there are some problems that need to be sorted out first.

The first issue is that the prime and double prime symbols are also commonly used minues/seconds and arcminutes/arcseconds.

The seconds issue is that single and double quotation marks, commonly used instead of prime symbols, obviously conflicts with "quotations" (for dates, symbols, etc.). It should be possible to work around this for simple expressions by assuming use of quotation marks as prime symbols if only one of each is found.

**Comment by svenstaro** [comment]
> The seconds issue is that single and double quotation marks, commonly used instead of prime symbols, obviously conflicts with "quotations" (for dates, symbols, etc.). It should be possible to work around this for simple expressions by assuming use of quotation marks as prime symbols if only one of each is found.

Sounds like an ok compromise to me.

**Comment by hanna-kn** [comment]
Done.

-  If expression contains degree symbol not part of temperature unit all quotation marks and prime symbols are used for arcminutes and arcseconds (not finished).
- If expression contains string of format `number"number'` (and no degree symbol), each quotation mark (and prime symbol) are interpreted as feet and inches. String using the above format is interpreted as feet _plus_ inches.
- If expression only contains at most one of each quotation mark (and no degree symbol), they (and prime symbols) are interpreted as feet and inches.
- If single quote/prime is interpreted as feet and nothing but a number (and possibly whitespace characters), double quote/prime (as inches) is appended.

**Comment by svenstaro** [comment]
Super nice!
